ui/core/ErrorText.tsx

```diff
@@ -0,0 +1,12 @@
+import { Text } from './Text'
+
+import styled from '@emotion/styled'
+import { css } from '@styled-system/css'
+
+export const ErrorText = styled(Text)(
+  css({
+    color: 'red.400',
+    py: 2,
+    px: 3,
+  })
+)
```

Comment:
Why extract this to a separate file?

Reply:
because it is used from `lib/pageWrappers/withValidPrivateKey.tsx` and `ui/core/Input.tsx` now (was exclusively used by input before that)